include/NovelRT/PluginManagement/PluginManagement.h

```diff
@@ -0,0 +1,28 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
+// for more information.
+
+#ifndef NOVELRT_PLUGINMANAGEMENT_H
+#define NOVELRT_PLUGINMANAGEMENT_H
+
+// PluginManagementDependencies
+#include "../Experimental/Graphics/Graphics.h"
+#include "../Experimental/Windowing/Windowing.h"
+
+/**
+ * @brief The NovelRT engine plugin system for loading modules such as Vulkan, GLFW3, OpenAL, and more.
+ */
+namespace NovelRT::PluginManagement
+{
+    class DefaultPluginSelector;
+    class IGraphicsPluginProvider;
+    class IWindowingPluginProvider;
+}
+
+// clang-format off
+#include "IGraphicsPluginProvider.h"
+#include "IWindowingPluginProvider.h"
+#include "TemporaryFnPtrs.h" //TODO: THIS HEADER IS A HACK - DELETE AFTER MVP
```

Comment:
We should probably open an issue to capture this work so we don't forget about it post-MVP